v1/jellyfin.yaml

```diff
@@ -16,15 +16,27 @@ instances:
       vendor-data: |
         package_update: true
         package_upgrade: true
-        
+
         packages:
         - apt-transport-https
-        
+
         runcmd:
-        - add-apt-repository universe
-        - curl -fsSL https://repo.jellyfin.org/ubuntu/jellyfin_team.gpg.key | gpg --dearmor -o /etc/apt/trusted.gpg.d/debian-jellyfin.gpg
-        - echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/ubuntu $( lsb_release -c -s ) main" | sudo tee /etc/apt/sources.list.d/jellyfin.list
-        - apt update -y
-        - apt install jellyfin -y
+        - sudo rm /etc/apt/sources.list.d/jellyfin.list
```

Comment:
Since this runs only on newly launched instances, `/etc/apt/sources.list.d/jellyfin.list` shouldn't exist and so this line is not necessary.

Reply:
Removed